Sources/Misc/StoreKitVersion.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  StoreKitVersion.swift
+//
+//  Created by Mark Villacampa on 4/13/23.
+
+import Foundation
+
+/// Defines which version of StoreKit may be used
+@objc(RCStoreKitVersion)
+public enum StoreKitVersion: Int {
```

Comment:
Have you considered removing `StoreKit2Setting`? The confusing part of this PR is that we now have both but since we're not using `.enabledOnlyForOptimizations`, we don't actually need that anymore?

Reply:
Bump